packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -768,13 +768,16 @@ export const autoJoinTeamsByOrgId = (parent: RootDataLoader) => {
 
       const pg = getKysely()
 
-      const teams = (await pg
-        .selectFrom('Team')
-        .where('orgId', 'in', verifiedOrgIds)
-        .where('autoJoin', '=', true)
-        .where('isArchived', '!=', true)
-        .selectAll()
-        .execute()) as unknown as Team[]
+      const teams =
+        verifiedOrgIds.length === 0
+          ? []
+          : ((await pg
```

Comment:
Found this while debugging, if there were no matching teams we would return a syntax error from PG instead of an empty array. Did not break anything as the caller used `.filter(isValid)`.